lobby/src/main/java/org/triplea/lobby/server/db/UserController.java

```diff
@@ -52,11 +52,27 @@ public boolean doesUserExist(final String username) {
   @Override
   public void createUser(
       final String name, final String email, final HashedPassword hashedPassword) {
+
+    try (Connection con = connection.get();
+        PreparedStatement ps =
+            con.prepareStatement("insert into user_role(id, name) values(10, 'ANONYMOUS')")) {
+
+      ps.execute();
+      con.commit();
+    } catch (final SQLException e) {
+      // ignore
```

Comment:
I think we should at least log it 🤷‍♂

Reply:
oh, this is a bit of a shameful hack. The `UserController` code is all dead code in super short order.
The reason the exception is ignored is because we are expecting a UK error indicating the value is already in DB. The only time we would expect that is while running tests, hence, the super-hack. I don't remember why I went for such an ugly approach, i think it's because the test-code was calling these methods for setup while testing the same logic. 

I added some TODOs and comments about this: 221069234